Header header

string text_to_say # Phrase to synthesize into speech
